packages/logging/src/index.js

```diff
@@ -452,13 +452,16 @@ Logging.prototype.getSinksStream = common.paginator.streamify('getSinks');
  * @resource [Log Overview]{@link https://cloud.google.com/logging/docs/api/ref_v2beta1/rest/v2beta1/projects.logs}
  *
  * @param {string} name - Name of the existing log.
+ * @param {object} options - Configuration object.
+ * @param {boolean} options.removeCircular - Replace circular references in an
```

Comment:
Sorry I didn't see this before, but for this line and where it's repeated, there's a grammar error: "in an logged objects"